package.json

```diff
@@ -98,7 +98,7 @@
     "less": "^3.12.2",
     "lint-staged": "^10.4.2",
     "log-symbols": "^4.0.0",
-    "node-abi": "^2.19.1",
+    "node-abi": "^2.19.3",
```

Comment:
I don't know if we actually need this direct dependency anymore. I don't believe it's actually referenced anywhere, and I've [fixed `electron-rebuild`](https://github.com/electron/electron-rebuild/pull/501) to not call `node-abi` unless there is actually a native module to rebuild.